### PR TITLE
Dev Add Cloud tools (cloudsplaining, cloudsploit and prowler)

### PR DESCRIPTION
**Pull request by clomic**
# Description
Add Cloud security assessment tools (cloudsplaining, cloudsploit and prowler)

### CODE REVIEW

File: sources/install/package_cloud.sh [hunk 1]
```diff
@@ -55,9 +55,38 @@ function install_scout() {
...
+    colorecho "Installing Cloudsplaining"
+    python3 -m pipx install cloudsplaining
+    add-history cloudsplaining
+    add-test-command "cloudsplaining --help"
+    add-to-list "cloudsplaining,https://github.com/salesforce/cloudsplaining,AWS IAM Security Assessment tool that identifies violations of least privilege and generates a risk-prioritized report."
+}
+
+function install_cloudsploit() {
+    colorecho "Installing Cloudsploit"
+    git -C /opt/tools/ clone https://github.com/aquasecurity/cloudsploit
```
[the remark this answers is not in the document]
Comment (clomic) [reply sources/install/package_cloud.sh]: Done (thks)

**Comment by QU35T-code** [comment]
@clomic,

Can you merge with the remote dev branch ? Otherwise, the update of the donpapi tool will block the workflow

File: sources/install/package_cloud.sh [hunk 2]
```diff
@@ -55,9 +55,38 @@ function install_scout() {
...
+    add-history cloudsplaining
+    add-test-command "cloudsplaining --help"
+    add-to-list "cloudsplaining,https://github.com/salesforce/cloudsplaining,AWS IAM Security Assessment tool that identifies violations of least privilege and generates a risk-prioritized report."
+}
+
+function install_cloudsploit() {
+    colorecho "Installing Cloudsploit"
+    git -C /opt/tools/ clone --depth 1 https://github.com/aquasecurity/cloudsploit
+    cd /opt/cloudsploit && npm install
+    add-aliases cloudsploit
```
[the remark this answers is not in the document]
Comment (clomic) [reply sources/install/package_cloud.sh:70]: done

File: sources/assets/zsh/aliases.d/cloudsploit [hunk 3]
```diff
@@ -0,0 +1 @@
+alias cloudsploit='/opt/cloudsploit/index.js'
```
[the remark this answers is not in the document]
Comment (QU35T-code) [reply sources/assets/zsh/aliases.d/cloudsploit:1]: > `/opt/tools/cloudsploit` (missing `tools`)

Not fixed.

File: sources/install/package_cloud.sh [hunk 4]
```diff
@@ -55,9 +55,38 @@ function install_scout() {
...
+    python3 -m pipx install cloudsplaining
+    add-history cloudsplaining
+    add-test-command "cloudsplaining --help"
+    add-to-list "cloudsplaining,https://github.com/salesforce/cloudsplaining,AWS IAM Security Assessment tool that identifies violations of least privilege and generates a risk-prioritized report."
+}
+
+function install_cloudsploit() {
+    colorecho "Installing Cloudsploit"
+    git -C /opt/tools/ clone --depth 1 https://github.com/aquasecurity/cloudsploit
+    cd /opt/cloudsploit && npm install
```
[the remark this answers is not in the document]
Comment (QU35T-code) [reply sources/install/package_cloud.sh]: > `/opt/tools/cloudsploit` (missing `tools`)

Not fixed.

**Review by QU35T-code — CHANGES_REQUESTED** [review]

**Comment by clomic** [comment]
Sorry, the directory was wrong (/opt/cloudsploit instead of /opt/tools/cloudsploit)...

**Comment by QU35T-code** [comment]
@clomic,

Can you check all my requests ? 
The directory is also wrong here : sources/assets/zsh/aliases.d/cloudsploit

**Comment by clomic** [comment]
Done

**Review by QU35T-code — APPROVED** [review]

**Review by ShutdownRepo — APPROVED** [review]

**Pull request merged**